app/assets/stylesheets/about.scss

```diff
@@ -0,0 +1,202 @@
+.menu-btn {
+  display: none;
+
+  &:checked ~ .dfe-vertical-nav {
+    display: block;
+  }
+
+  &:not(:checked) ~ .dfe-vertical-nav {
+    display: none;
+
+    @include govuk-media-query($from: tablet) {
+      display: block;
+    }
+  }
+}
+
+label.menu-icon {
+  display: flex;
+  align-items: center;
+  font-weight: bold;
+  gap: govuk-spacing(2);
+  color: $govuk-link-colour;
+
+  @include govuk-media-query($from: tablet) {
+    display: none;
+  }
+}
+
+.dfe-vertical-nav {
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 1.14286;
+}
+
+.dfe-vertical-nav__section {
+  margin: 0 0 govuk-spacing(4);
+  padding: 0px;
+  list-style-type: none;
+}
+
+.dfe-vertical-nav__section-item {
+  border-left: 4px solid govuk-colour('mid-grey');
+  font-size: 16px;
+  line-height: 1.25;
+
+  &:hover {
+    border-left: 4px solid $color_dfe-secondary-blue;
+  }
+
+  &--current {
+    border-left: 4px solid $color_dfe-blue;
+    font-weight: 700;
+    background: govuk-colour('light-grey');
+
+    .dfe-vertical-nav__link {
+      border-left-color: $color_dfe-blue;
+      font-weight: 700;
+      color: $color_dfe-blue;
+
+      &:hover {
+        text-decoration: none;
+      }
+    }
+  }
+
+}
+
+.dfe-vertical-nav__link {
+  display: block;
+  padding: 7px 30px 8px 10px;
+  text-decoration: none;
+  margin-bottom: govuk-spacing(1);
+  color: $color_dfe-blue;
+
+  &:active,
+  &:hover {
+    color: govuk-colour('blue');
+    border-left-color: $color_dfe-secondary-blue;
+    box-shadow: none;
+    outline: 0;
+  }
+
+  &:focus {
+    background: govuk-colour('yellow');
+    color: govuk-colour('black');
+    box-shadow: none;
+    outline: 0;
+  }
+}
+
+.about-module-box {
+  background: $light-blue;
+  display: flex;
+  flex-direction: column-reverse;
+  margin-bottom: govuk-spacing(6);
+
+  @include govuk-media-query($from: tablet) {
+    flex-direction: row;
+  }
+
+  &-panel {
+    flex: 100%;
+    padding: 1.5em;
+    background-size: cover;
+    background-position: center;
+
+    @include govuk-media-query($from: tablet) {
+      flex: 50%;
+      padding: 3em;
+    }
+  }
+}
+
```

Comment:
Please add the following:

.about-module-box-panel p:last-child {
    margin-bottom: 0;
}